packages/search/README.md

```diff
@@ -1,31 +1,86 @@
 # Search
 
-A search input component.
+The search module.
 
 ## Installation
 
-Install the component.
+Install the package.
 
 ```bash
 yarn add @automattic/search
 ```
 
 ## Internationalization
 
-This package depends directly on `@wordpress/i18n` for translations. This means consumers must provide locale data prior to using the component.
+This package depends directly on `@wordpress/i18n` for translations. This means consumers must provide locale data prior to using it.
 
 ## Development Workflow
 
 This package is developed as part of the Calypso monorepo. Run `yarn`
 in the root of the repository to get the required `devDependencies`.
 
-### Using [Storybook](https://storybook.js.org/)
+## Using [Storybook](https://storybook.js.org/)
+
+`yarn workspace @automattic/search run storybook`
+
+## What's included?
 
-`yarn workspace @automattic/components run storybook`
+### Search input
 
-### Search Modes
+A search input component.
+
+#### Search Modes
 
 There are 2 search modes that can be used through the `searchMode` prop:
 
 - `when-typing` (default): The search component calls the `onSearch` prop while the user is typing.
 - `on-enter`: The search component only triggers the `onSearch` prop when the user hits the `Enter` key.
+
+### `useFuzzySearch`
+
+A hook that exposes fuzzy searching functionality.
+
+It can be beneficial especially as the size of the dataset grows because this mechanism does not walk through the whole list. Also, some users might want to find an object that they do not remember the exact name, so entering a partial string, or even one with typos, would still return results ranked by proximity.
```

Comment:
```suggestion
`useFuzzySearch` can be beneficial especially as the size of the dataset grows because this mechanism does not walk through the whole list. Also, some users might want to find an object that they do not remember the exact name, so entering a partial string, or even one with typos, would still return results ranked by proximity.
```

Reply:
This suggested change looks like it might not apply directly...